x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers.ts

```diff
@@ -145,8 +145,8 @@ function getSpanItem(span: Span): IWaterfallItemSpan {
 
 export function getClockSkew(
   item: IWaterfallItem,
-  parentItem: IWaterfallItem | undefined,
-  parentTransactionSkew: number
+  parentTransactionSkew: number,
+  parentItem?: IWaterfallItem
```

Comment:
this function really just needs the `parentItem` at this point, right?

Down on line 160 you even reference it as `parentItem.skew`, could you do the same on line 153 and remove `parentTransactionSkew` altogether?

Reply:
I don't think so. `parentItem` can both be a span and a transaction. `parentTransactionSkew` is for the parent transaction (which is not necessarily the parent).

Maybe we can simplify it though.

Reply:
... or you might actually be right.

Reply:
Remove `parentTransactionSkew`. Good idea!